portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,30 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+<meta charset="UTF-8">
+<title>Roland's - Portfolio</title>
+<link rel="stylesheet" href="style.css">
+<script src="script.js"></script>
```

Comment:
It looks like you're removing the <html> and <head> here?  You should keep those, and lines 2-5 here should be in the head

Reply:
So according to the css and html guide you shared with me [(section 3)](https://google.github.io/styleguide/htmlcssguide.html#HTML_Style_Rules), it highlights that you can omit the tags I deleted because they are optional tags

Reply:
> It looks like you're removing the and here? You should keep those, and lines 2-5 here should be in the head

So according to the css and html guide you shared with me [(section 3)](https://google.github.io/styleguide/htmlcssguide.html#HTML_Style_Rules), it highlights that you can omit the tags I deleted because they are optional tags